test/lib/rule-finder.js

```diff
@@ -77,57 +85,107 @@ describe('rule-finder', function() {
 
   it('specifiedFile (relative path) is passed to the constructor', function() {
     var ruleFinder = getRuleFinder(specifiedFileRelative)
-    assert.deepEqual(ruleFinder.getUnusedRules(), ['baz-rule', 'react/bar-rule', 'react/baz-rule', 'react/foo-rule'])
+    assert.deepEqual(ruleFinder.getUnusedRules(), [
+      'baz-rule',
+      'react/bar-rule',
+      'react/baz-rule',
+      'react/foo-rule',
+      'something/two-rule',
```

Comment:
May not be a practical use case, curious to know, how does `eslint` handle, two or more scoped plugins, of the same name?